Jenkinsfile

```
@@ -997,6 +1001,25 @@ pipeline {
                                                         needs_gpu:false, needs_reboot:false, build_install: "true")
                     }
                 }
+                stage('dbsync gfx942') {
```

```suggestion
                stage('Dbsync gfx94X') {
```
or something similar where the first letter is upper case to maintain consistency.